CMake/CppcheckTargets.cmake

```diff
@@ -280,35 +255,24 @@ function(add_cppcheck _name)
     # --------------------------------------------------------------
     set(_cppcheck_compile_args --check-config ${_cppcheck_include} ${_cppcheck_def})
 
-     itk_add_test(
-       NAME
-         ${_name}CPPCheckTest
-       COMMAND
-         "${CPPCHECK_EXECUTABLE}"
-         ${CPPCHECK_TEMPLATE_ARG}
-         ${_cppcheck_args}
-         ${_cppcheck_compile_args}
-         ${_files})
-
-      set_tests_properties(${_name}CPPCheckTest
-        PROPERTIES
-        FAIL_REGULAR_EXPRESSION
-        "${CPPCHECK_FAIL_REGULAR_EXPRESSION}")
-
-      add_custom_command(TARGET
-        all_cppcheck
-        PRE_BUILD
-        COMMAND
-          ${CPPCHECK_EXECUTABLE}
-          ${CPPCHECK_QUIET_ARG}
-          ${CPPCHECK_TEMPLATE_ARG}
-          ${_cppcheck_args}
-          ${_files}
-        WORKING_DIRECTORY
-          "${CMAKE_CURRENT_SOURCE_DIR}"
-        COMMENT
-          "${_name}_cppcheck: Running cppcheck on target ${_name}..."
-       V ERBATIM)
```

Comment:
`V ERBATIM` with space seems like a bug. Does it affect anything?

Reply:
This is an unrelated issue: patch here: https://github.com/InsightSoftwareConsortium/ITK/pull/4234